public/app/percona/add-instance/components/AzureDiscovery/components/Instances/InstancesColumns.tsx

```diff
@@ -0,0 +1,74 @@
+import { Button } from '@grafana/ui';
+import React from 'react';
+import { DATABASE_LABELS, Databases } from 'app/percona/shared/core';
+import { styles } from './Instances.styles';
+
+const getEngineType = (type?: string) => {
+  switch (type) {
+    case 'DISCOVER_AZURE_DATABASE_TYPE_MYSQL':
+      return DATABASE_LABELS[Databases.mysql];
+    case 'DISCOVER_AZURE_DATABASE_TYPE_MARIADB':
+      return DATABASE_LABELS[Databases.mariadb];
+    case 'DISCOVER_AZURE_DATABASE_TYPE_POSTGRESQL':
+      return DATABASE_LABELS[Databases.postgresql];
+    case 'DISCOVER_AZURE_DATABASE_INVALID':
+      return 'Unknown type';
+    default:
+      return 'Unknown type';
+  }
+};
+
+const getDatabaseType = (type?: string) => {
+  switch (type) {
+    case 'DISCOVER_AZURE_DATABASE_TYPE_MYSQL':
+    case 'DISCOVER_AZURE_DATABASE_TYPE_MARIADB':
+      return Databases.mysql;
+    case 'DISCOVER_AZURE_DATABASE_TYPE_POSTGRESQL':
+      return Databases.postgresql;
+    default:
+      return '';
+  }
+};
+
+// @ts-ignore
```

Comment:
There's types for this two variables so no need to ignore

Reply:
Done